src/OnlineRetailPortal.Mock/Models/Price.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace OnlineRetailPortal.Mock
+{
```

Comment:
Remove all the duplicate models declared in mock project.. 
why do we need them?

Reply:
I have created these models bcoz we are creating a List of Products in mock.